frontend/dynamic-demo-plugin/locales/en/dynamic-demo-plugin.json

```diff
@@ -0,0 +1,3 @@
+{
+    "Subject": "Subject"
```

Comment:
Is this used anywhere in the demo plugin?

I would only add the JSON files if we're actually using it somewhere in the demo plugin to give a real example.

Reply:
no its not, in that case I can just put there and empty json

Reply:
We can have a follow-up PR that illustrates localization usage in dynamic demo plugin.

There should be two examples:
- localizing strings in `console-extensions.json`
- localizing strings in a plain function
- localizing strings in a React component

Once this PR is merged and the conventions are settled, I can write that PR.